### PR TITLE
Add .yaydoc.yml and fix markdown

### LINKED ISSUE #817: Please create a simple website without Jekyll with Material Design
Please create a simple website in the same design as the app itself, e.g. something simple like https://assistant.google.com

We should use Yaydoc for this. So, the Readme.md is converted into a startpage. Simply make the start page into a good resource.

### PR DESCRIPTION
**Pull request by imujjwal96**
Fix #817 

Changes:
Added .yaydoc.yml and fixed markdown

Screenshots for the change: 
http://yaydoc.herokuapp.com/preview/ujjwalb1996@gmail.com/d01e95ef-6051-492a-8860-0b2d6622656e_preview/
image
image

### CODE REVIEW

**Comment by imujjwal96** [comment]
@mariobehling  @pa1pal 
@pri22296 @sch00lb0y  Please review

File: .yaydoc.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+metadata:
+  author: FOSSASIA
+  projectname: Phimp.me
+  version: development
+build:
+  theme: sphinx_fossasia_theme
+  source: docs
+  logo: _static/logo.png
```
Comment (pri22296) [inline .yaydoc.yml]: Since the logo is already being displayed in the README, I guess we don't really need this.

File: README.md [hunk 2]
```diff
@@ -34,47 +33,43 @@ We have the following branches
...
-     
-    
-</td>
-    </tr>
-  </table>
-  
-   
+<table>
+  <tr>
+    <td><img src="/docs/_static/screenshots/screenshot_1.png"></td>
```
Comment (pri22296) [inline README.md]: Any reason for text going italics from here?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,9 @@
 # phimpme-android
-<img src="/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png" align="left" width="200" hspace="10" vspace="10">
+![Phimpme](/docs/_static/logo.png)
```
Comment (pri22296) [inline README.md]: `_static` is only required when referring from embedded html. Not required when using markdown's image reference syntax.

File: README.md (hunk 2 above)
Comment (imujjwal96) [inline README.md]: i didn't. Rendered as is

**Comment by imujjwal96** [comment]
@pri22296  http://yaydoc.herokuapp.com/preview/ujjwalb1996@gmail.com/a2fce70a-37c0-4404-b548-74647c4579d0_preview/ The logo is not rendered

File: README.md (hunk 3 above)
Comment (pri22296) [inline README.md]: I think it's still required that the file referred be in `docpath`. Not sure though.

Comment (imujjwal96) [inline README.md]: Now what?

**Comment by pa1pal** [comment]
@imujjwal96 is the Link down? I got not found.

**Comment by imujjwal96** [comment]
Yes! we do not have a persistent storage for the app. You may however generate one to review. 
Use: https://github.com/imujjwal96/phimpme-android.git as the GITURL

File: .yaydoc.yml (hunk 1 above)
Comment (imujjwal96) [inline .yaydoc.yml]: It is not just about the start page. What about when we will have multiple pages?

Comment (pri22296) [inline .yaydoc.yml]: Good point. Then keep the logo.

**Comment by pa1pal** [comment]
@imujjwal96 ok, I will generate.

**Comment by imujjwal96** [comment]
@pri22296 Has there been any breaking change at yaydoc that will require us to update this PR?

**Comment by pri22296** [comment]
No, looks fine. Just move the image referred from the README to docs/

**Comment by imujjwal96** [comment]
@pri22296 The logo?

File: README.md (hunk 3 above)
Comment (pri22296) [inline README.md]: @imujjwal96 This one. The image doesn't need to be inside `docs/_static`, but it does need to be inside `docs`

Comment (imujjwal96) [inline README.md]: With this fix: https://github.com/fossasia/yaydoc/pull/298; we don't need to care about the location, right?

Comment (pri22296) [inline README.md]: Honestly, I'm not sure if I'd be able to fix this issue. I'm trying a lot of things but didn't have any good results till now. Better not depend on it.

Comment (imujjwal96) [inline README.md]: :worried:  Ok. then!

**Comment by pa1pal** [comment]
@imujjwal96 can we merge this?

File: .yaydoc.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+metadata:
+  author: FOSSASIA
+  projectname: Phimp.me
+  version: development
+build:
+  theme: sphinx_fossasia_theme
```
Comment (pri22296) [inline .yaydoc.yml]: @imujjwal96 Please update
```yaml
theme: sphinx_fossasia_theme
```
to
```yaml
theme:
  name: sphinx_fossasia_theme
```
before this gets merged.

**Comment by imujjwal96** [comment]
@pri22296  @pa1pal  Updated

**Comment by mariobehling** [comment]
@imujjwal96 Thank you. 

Next steps deploy it to phimp.me using gh-pages branch.

**Pull request merged**